glide.yaml

```diff
@@ -30,7 +30,7 @@ import:
   - thrift
 
 - package: gopkg.in/vmihailenco/msgpack.v2
-  version: a1382b1ce0c749733b814157c245e02cc1f41076
+  version: f4f8982de4ef0de18be76456617cc3f5d8d8141e
```

Comment:
We should probably verify by deploying this in an old cluster and upgrading in place.

The raw data blocks are encoded in msgpack too so it might break normal block bootstrapping (outside of commit log bootstrapper). 

This is 100% easily testable though thankfully.